src/bacnet/basic/object/time_value.h

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file
+ * @author Mikhail Antropov
+ * @date June 2023
+ * @brief Color object, customize for your use
```

Comment:
please update the JavaDoc header to represent the Time Value object, not color object.